pilot/cmd/pilot-agent/status/server.go

```diff
@@ -493,10 +493,16 @@ func (s *Server) handleStats(w http.ResponseWriter, r *http.Request) {
 	var envoyCancel, appCancel context.CancelFunc
 	defer func() {
 		if envoy != nil {
-			envoy.Close()
+			err = envoy.Close()
+			if err != nil {
+				log.Infof("envoy connection is not closed: %v", err)
```

Comment:
not sure how this could be error

Reply:
this wouldn't be an issue for us, just we have some places where we were not handling the errors. Added some info messages from them.